build.sbt

```diff
@@ -312,6 +324,18 @@ lazy val ogcExample = (project in file("ogc-example"))
       scalatest,
       decline
     ),
+    libraryDependencies ++= {
```

Comment:
I will remove that once Jenkins would publish this commit: https://github.com/locationtech/geotrellis/commit/fe437e2a12d3cdf288a014fce206698040dfa9d4